index.js

```diff
@@ -0,0 +1,76 @@
+const uuid = require('node-uuid');
+
+/**
+ * Filter for response messages
+ * @param  {object}  msg - message to check
+ * @return {Boolean}
+ */
+function isChildMessage(msg) {
+  return Boolean(this && this.header && msg && msg.parent_header &&
+    this.header.msg_id === msg.parent_header.msg_id);
+}
+
+/**
+ * Create a message
+ * @param  {string} username
+ * @param  {string} session  guid for the session
+ * @param  {string} msg_type
+ * @return {object}          msg
+ */
+function createMessage(username, session, msg_type) {
```

Comment:
Would we assume this always creates the latest version of the Jupyter messaging protocol?